# Scientific sprint for better eukaryotic transcriptome annotation

Transcriptome annotation is a milestone for all functional genetic studies. But for several reasons, we do not have a consensus A-Z annotation pipeline.  Diverse research questions, different biological, computational and financial resources, and rapidly evolving technologies are among a long list of reasons. We can see how major genome annotation databases (e.g. RefSeq and ensemble) adopt different annotation pipelines which has significant impact on down stream analyses ([Zhao and Zhang. BMC Genomics. 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4339237/)). The problem has exponential impact when different research labs make their own annotations with their preferred software packages, and their own working parameters. It is too idealistic to think about single pipeline that can address all questions in all domains but more realistically we can learn from each other to make them better.

Crowd sourcing and democratization of research are among the important - if not the most important – sources of power and achievements made by open science. Several examples have proven the power of working together but we need to uncover the power of “thinking” together.  This is the main idea behind this work. Can we think together about making transcriptome annotation more efficient? Can we – as a community – put our thoughts together in one article that we can continuously update? This is possible and will not only enhance the field of transcriptome annotation but also set up a working example that can be expanded upon by researchers with different scopes of questions and promote the open source discussion. 

Let’s start by defining the exact terms discussed here and highlighting the scope of the forthcoming article as well as our suggested working plan and who can share:

**Transcriptome annotation:** This is large-scale prediction of gene structure and/or function. Gene here is defined as any transcribed genomic locus either coding or non-coding.  This definition is to emphasis the inclusion of non-coding transcription and also to ensure the exclusion of all other aspects of genome annotations e.g. genome architecture and organization, regulatory sequences, epigenetic landmarks, etc.

**Scope of the article:** Although this article should highlight significant research articles and software packages in the filed, it is NOT just a literature review. We are gathering here to share experiences, frustrations, doubts and suggestions in this daily growing field.   

**Working plan:** We will have a separate Github issue for each step to organize the work cycle:

1.	Prepare a list of all contributors
2.	Share ideas and brainstorm about all interesting questions related to the topic. I can think of two main categories of ideas. 
  1. Literature: Topics that we should cover by comprehensive literature search
  2. Discussion: Experiences, frustrations, doubts and suggestions
3.	Select editorial board from all contributors (I suggest 3 members; one leader and 2 co-leaders; one for reference based transcriptome assembly and another one for denovo assembly). 
4.	The editorial board will lead a discussion about the outline of the 1st draft and assign tasks to the contributors.
5.	Each contributor will lead an issue for his task where he/she shares his/her initial draft paragraph and get feed back from all of us. The issue will be announced closed by final draft of this paragraph.
6.	The editorial board should collect all draft paragraphs into a first draft version, and start an open discussion about suggested changes. 
7.	The board will perform the necessary changes (or assign them to contributors), and publish the first draft in the bioRxiv archive.
8.	Ask the community for feedback and may be repeat step 3-5 to make a second version.
9.	The editorial board will produce a final curated version for peer review journal

The editorial board can change all these suggestions after group discussion. I am expecting this to be done in a motivational friendly environment, making an inspiring example for an open science model of collaboration. 

**Who can share?** If you read thought until this point, then definitely you can share in this work. We will have a lot of discussions, literature search, scripting, and writing enough for everyone. The editorial board should try to get all contributors involved then judge their commitment and contribution        

**Time Plan:** 8 weeks plan

* 1 week for discussions and brainstorming.
* 1 week for the editorial board to lead a discussion about the outline of the 1st draft and assign tasks to the contributors
* 1 week for the contributors to finish their parts
* 1 week for the editorial board to prepare 1st draft
* 1 week for edits and updates before publishing in the bioRxiv archive
* 3 weeks for community discussion and editorial board preparation peer reviewed version

**Disscusion plateform:** 
We have started [our own instance of Discourse](https://reviews.oxli.org/) for disscusions. Thanks for [Luiz Irber](https://github.com/luizirber) who volunteered to install the plateform in the amazon cloud and thanks for [C. Titus Brown](https://github.com/ctb) who generously accepted to fund the website. Discourse is an open source API that incorportes many feature making community disscusions more useful. You can follow the [software link](http://www.discourse.org/) to know more about the features of Discourse. 

**What should I do now?** Please go ahead and open the [“contributors”](https://github.com/dib-lab/TransAnn-Review/issues/1) issue to add your name and email. Next you can open the issue [“Brainstorm”](https://github.com/dib-lab/TransAnn-Review/issues/2) to find out how to join our disscusion plateform. 

If you have any questions or suggestions about the project to be more productive, please drop them in the [Suggestion box](https://github.com/dib-lab/TransAnn-Review/issues/3).

